homeassistant/components/synology/__init__.py

```diff
@@ -1 +1,76 @@
 """The synology component."""
+
+import logging
+
+import requests
+from synology.surveillance_station import SurveillanceStation
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_TIMEOUT,
+    CONF_URL,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+import homeassistant.helpers.config_validation as cv
+
+from .const import (
+    DATA_NAME,
+    DATA_SURVEILLANCE_CLIENT,
+    DATA_VERIFY_SSL,
+    DEFAULT_NAME,
+    DEFAULT_TIMEOUT,
+    DEFAULT_VERIFY_SSL,
+    DOMAIN,
+    DOMAIN_DATA,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler

Reply:
Can't this be exempted from the ADR since it was done before the decision date. Seems like it was just waiting to be merged back in March.

Reply:
No, we started the PR review after the ADR was announced.

Reply:
Hi @MartinHjelmare . I have resolved the merge conflicts. I will only be able to pick up the rest at the beginning of June. 
However there seems to be another PR (https://github.com/home-assistant/core/pull/35821) that adds the config flow to the Synology integration. If that one goes forward in the meantime, I will gladly add the switch later.

Reply:
Ok. If you don't plan to work here for a while, we'd appreciate if you close. Then you can reopen later and we'll keep the conversation history intact.

We try to decrease our open PR buffer. Thanks!